query-engine/driver-adapters/executor/src/rn.ts

```diff
@@ -0,0 +1,83 @@
+import Axios from "axios";
+
+export function createRNEngineConnector(
+  url: string,
+  schema: string,
+  logCallback: (msg: string) => void
+) {
+  const port = "3000";
+  const baseIP = "192.168.178.20";
+  const deviceUrl = `http://${baseIP}:${port}`;
+  const axios = Axios.create({
+    baseURL: deviceUrl,
+    headers: {
+      "Content-Type": "application/json",
+    },
+    transformResponse: (r) => r,
+  });
```

Comment:
Err, it's a mess. We've got two urls to deal with, the db url and the emulator URL, and there's no good way to pass it around without stretching the existing code.

I'd suggest you make that URL gotten from the env until we refactor that to use Rust instead of JS.

Reply:
Something like `PRISMA_MOBILE_EMULATOR_URL`. I hate it, but it's better than a hardcoded URL.